invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/components/Document/DocumentAuthors.js

```diff
@@ -9,20 +9,23 @@ export class DocumentAuthors extends Component {
     return (
       <div className="document-authors-list-wrapper">
         {prefix ? prefix + ' ' : null}
-        <List horizontal className={'document-authors-list'}>
-          {metadata
-            ? metadata.authors.map((author, index) => (
+          {metadata && metadata.authors
+            ?
```

Comment:
Would be easier to read the code with `{metadata && metadata.authors &&` instead of `?` since it's `null` if not.